.gitignore

```diff
@@ -11,3 +11,4 @@
 /externals/3rdparty/winpcap/Include
 /externals/3rdparty/winpcap/Lib
 
+.idea
```

Comment:
You are missing a LF here (nice to add the .idea folder, might be useful for some :) )

Reply:
@christophe-calmejane you mean add to this repo?

Reply:
Not specifically but if someone wants to use IntelliJ, the folder is already ignored (I mean, it doesn't hurt).